### DISCUSSION TITLE
Send data with JSON format to the server - Parse JSON to HTML in editor

### DISCUSSION THREAD

**Discussion by aspyropoulos** [ueberdosis/tiptap#1640] [General Questions] [answered]
I am using tiptap with Livewire and Alpinejs. I followed the instructions [here](https://www.tiptap.dev/installation/livewire) for installation. 

I've seen relevant discussion threads but I am confused. There is this [generate-html-from-json](https://www.tiptap.dev/api/utilities/html/#generate-html-from-json) package but there is the [editor.getJSON, editor.getHTML](https://www.tiptap.dev/guide/output/#option-2-html) out of the box.

Considering the above stack what is the best way to achieve html to json conversion and vice versa. Could be an option, upon saving to use the `editor.getJSON` in a separate variable or?

**Comment by Priestch** [comment, marked as answer]
@aspyropoulos The inner data model of editor is always json like, but you can create editor content from html or json, and you can get and save editor content use `editor.getJSON` or `editor.getHTML`. All depends on the data format you want to load and save.

**Reply by aspyropoulos** [reply]
Ah I see. I end up using the `editor.getJSON` upon saving of my form and worked really good. Thanks
